paper/paper.md

```diff
@@ -32,72 +32,72 @@ bibliography: paper.bib
+`mechkit` **Why capitalized?** contains methods and operators
+that facilitate common tasks within tensor algebra and tensor notation.
 
-In the context of engineering applications within three spatial dimensions, deformations and stresses in solids are
+In the context of engineering applications in three spatial dimensions, deformations and stresses in solids are
 usually described by second-order tensors.
 As linear mappings between observed deformations and possibly causal stresses,
-fourth-order tensors have a special task in the field of linear elasticity.
-To this end, the methods in `mechkit` are primarily focussed on second- and fourth-order tensors. Main motivations can thus be found in the research in
+fourth-order tensors have a special role in the field of linear elasticity.
+To this end, the methods in `mechkit` are primarily focussed on second- and fourth-order tensors. Main motivations can thus be found in the research concerning
 linear elasticity
 [@Bertram2015], [@Mandel1965], [@Fedorov1968], [@Mehrabadi1990], [@Thomson1856],
 [@Cowin1992], [@Rychlewski2000], [@Spencer1970], [@Boehlke2001], [@Brannon2018]
-and the description of microstructures of fiber reinforced
+and the description of microstructures of fiber-reinforced
 composite materials
 [@Bauer2021], [@Kanatani1984], [@Advani1987].
 
-The implementations in `mechkit` aim at usability, seek to provide understandable source code
+The implementations in `mechkit` aim at usability, seek to provide understandable source code,
 and do not put major emphasis on performance.
 Furthermore, the implementations follow, as directly as possible,
 the notation and formulation in the respective scientific sources.
 A redundant implementation of identical operations based on different
-sources is strived for.
-
+sources is strived for. **For validation or just for fun?**
 
 # Statement of need
 
-The methods and operators occurring in the field of linear elasticity and material modeling are
-comparatively simple and compact.
-Due to this simplicity, they are usually implemented independently by each scientist
-during the studies and
-do not find their way into more general libraries which are common in other research fields such as physics or computer science.
-Apart from didactic advantages, this procedure causes problems for
+The methods and operators for the description of linear elasticity and material modeling are mathematically simple and can be expressed in a compact manner.
+Due to this relative simplicity, they are usually implemented independently by each scientist for use in their studies.
+Consequentially, unlike in other research field such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
+Apart from didactic advantages **I don't get what you want to say here. Suggestion in line 65.**, this procedure causes problems for
```

Comment:
Thanks for feedback, let's talk about this, but please remove your changes.
Why are all these lines highlighted, although you did just add a comment?
Please do not change line endings.